scalability_and_performance/enabling-workload-partitioning.adoc

```diff
@@ -6,57 +6,34 @@ include::_attributes/common-attributes.adoc[]
 
-The minimum number of reserved CPUs required for the cluster management is four CPU Hyper-Threads (HTs).
-With workload partitioning, you annotate the set of cluster management pods and a set of typical add-on Operators for inclusion in the cluster management workload partition.
-These pods operate normally within the minimum size CPU configuration.
-Additional Operators or workloads outside of the set of minimum cluster management pods require additional CPUs to be added to the workload partition.
+In the context of enabling workload partitioning and managing CPU resources effectively, nodes within the cluster must identify themselves as supporting a new resource type called `management.workload.openshift.io/cores` and accurately report their CPU capacity. Workload partitioning can be enabled during cluster installation only by adding the additional field: `cpuPartitioningMode` to the `install-config.yaml` file.
 
-Workload partitioning isolates user workloads from platform workloads using standard Kubernetes scheduling capabilities.
+When workload partitioning is enabled, the `management.workload.openshift.io/cores` resource allows the scheduler to correctly assign pods based on the cpushares capacity of the host, not just the default cpuset. This ensures more precise allocation of resources for workload partitioning scenarios.
 
-The following changes are required for workload partitioning:
+Workload partitioning respects CPU requests and limits defined on pods. From 4.16 accurate CPU usage limitations are set for platform pods utilizing CPU partitioning. Resource requests, rather than limits, are the primary value in setting accurate CPU usage limitations. 
 
-. In the `install-config.yaml` file, add the additional field: `cpuPartitioningMode`.
-+
-[source,yaml]
-----
-apiVersion: v1
-baseDomain: devcluster.openshift.com
-cpuPartitioningMode: AllNodes <1>
-compute:
-  - architecture: amd64
-    hyperthreading: Enabled
-    name: worker
-    platform: {}
-    replicas: 3
-controlPlane:
-  architecture: amd64
-  hyperthreading: Enabled
-  name: master
-  platform: {}
-  replicas: 3
-----
-<1> Sets up a cluster for CPU partitioning at install time. The default value is `None`.
-+
-[NOTE]
-====
-Workload partitioning can only be enabled during cluster installation. You cannot disable workload partitioning postinstallation.
-====
-
-. In the performance profile, specify the `isolated` and `reserved` CPUs.
-+
-.Recommended performance profile configuration
+If, for example, a request is set to 20 and the limit to 40, the scheduler prioritizes the lower value to allocate resources according to actual usage. Requesting 20 represents the typical need, while 40 is the upper limit. Prioritizing requests over limits optimizes resource allocation, aligning with usage needs and avoiding excess provisioning.
```

Comment:
Is this only relevant to to platform Pods?
If so, why is this relevant to the end user, they are not supposed to create those

Reply:
This might be too much detail, we're essentially describing how the scheduler works here. 

The only thing I would want to make mention of is something I wrote above regarding the custom resource values equaling the same for requests and limits. That's a requirement of Kubernetes

https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#consuming-extended-resources
> Note: Extended resources cannot be overcommitted, so request and limit must be equal if both are present in a container spec.